CMakeLists.txt

```diff
@@ -108,14 +108,6 @@ find_package(GLM REQUIRED)
 find_package(LibSndFile REQUIRED)
 find_package(SDL2 REQUIRED)
 
-include_directories(
-    ${SDL2_INCLUDE_DIR}
-    ${GLM_INCLUDE_DIRS}
-    ${OPENGL_INCLUDE_DIR}
-    ${BULLET_INCLUDE_DIRS}
-    ${OPENAL_INCLUDE_DIR}
-)
```

Comment:
This commit also feels like more build changes which are not related to VS2017, they should probably be a seperate PR